index.html

```diff
@@ -5,6 +5,10 @@
   <body>
     <h1> Physician. Entrepreneur. Teacher.</h1>
     <img src= "https://avatars2.githubusercontent.com/u/25405859?v=4>"
```

Comment:
The following code has a slight typo where the closing parentheses is outside of the attribute value. Please retry with the following :)
 
`<img src= "https://avatars2.githubusercontent.com/u/25405859?v=4">`